neuraxle/steps/numpy.py

```diff
@@ -80,36 +82,92 @@ def _concat(self, data_inputs):
         return np.concatenate(data_inputs, axis=self.axis)
 
 
-class NumpyConcatenateInnerFeatures(NumpyConcatenateOnCustomAxis):
+class NumpyConcatenateOnCustomAxis(metaclass=DeprecatedMetaClass):
```

Comment:
Should we do this for the insinstance to work ? 

```suggestion
class NumpyConcatenateOnCustomAxis(metaclass=DeprecatedMetaClass, NumpyConcatenateOnAxis):
```

Reply:
No the isinstance method is handled correctly by the DeprecatedMetaClass.